src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Triage and run diagnostics in eight clicks
+tags:
+  - APM
+  - Triage
+  - Diagnostics
+metaDescription: Use our application monitoring tools to quickly triage and run diagnostics during an incident.
+---
+import apmCheckoutServiceinApm from 'images/apm_screenshot-full_checkout-service-in-apm.png'
+
+import apmTimePickerinApm from 'images/apm_screenshot-full_time-picker-in-apm.png'
+
+import accountsSelectSlowestTransaction from 'images/accounts_screenshot-crop_select-slowest-transaction.gif'
+
+import apmInvestigateThroughputinAPM from 'images/apm_screenshot-full_investigate-throughput-in-APM.gif'
+
+import apmTransactionDetailsPage from 'images/apm_screenshot-crop_Transaction-details-page.png'
+
+import apmTransactionBreakdown from 'images/apm_screenshot-crop_transaction-breakdown.png'
+
+import apmThroughputTransactionsDetailsPage from 'images/apm_screenshot-crop_throughput-transactions-details-page.gif'
+
+import apmTraceDetails from 'images/apm_screenshot-crop_trace-details.png'
+
+import apmTransactionTracesinAPM from 'images/apm_screenshot-full_transaction-traces-in-APM.gif'
+
+import apmTransactionsTabinApmOverview from 'images/apm_screenshot-full_transactions-tab-in-apm-overview.gif'
+
+
+We all know what it's like to wake up in the morning to a ping from your leadership team telling you that something is going wrong and they need it fixed as soon as possible. This is where our APM tools come in. When you have limited time and increased urgency, you can use APM to run diagnostics and triage issues in eight clicks. 
```

Comment:
we sometimes have an issue at NR where we redundantly describe the same features/benefits. Basically: almost all our tools can be said to "help you run diagnostics and triage issues," across the board. The test that I apply is: could I say this same thing in many different docs? if that's the case, I might try to find something a bit more focused on the specific features. Something about mentioning APM agent-monitored applications/services specifically, and the specific UI pieces you use. I realize it's an intro and should be short, but that's what I'd be aiming for. Something that is unique to the feature & doesn't apply to many NR features.

Reply:
this is a good callout, thanks!